lstchain/reco/utils.py

```diff
@@ -200,12 +200,12 @@ def reco_source_position_sky(cog_x, cog_y, disp_dx, disp_dy, focal_length, point
     sky frame: `astropy.coordinates.sky_coordinate.SkyCoord`
     """
     src_x, src_y = disp.disp_to_pos(disp_dx, disp_dy, cog_x, cog_y)
-    return camera_to_sky(src_x, src_y, focal_length, pointing_alt, pointing_az)
+    return camera_to_altaz_frame(src_x, src_y, focal_length, pointing_alt, pointing_az)
 
 
-def camera_to_sky(pos_x, pos_y, focal, pointing_alt, pointing_az, time = None):
+def camera_to_altaz_frame(pos_x, pos_y, focal, pointing_alt, pointing_az, time = None):
```

Comment:
The frame in the end is a little strange (Either its `camera_frame_to_altaz_frame` or `camera_to_altaz`).
I would just leave out the frame.

`time` should be `obstime` to be consistent with the astropy API, and then you also don't need the `if` statement below.

Reply:
Thanks @MaxNoe for your usefull input.
Here I disagree with you but may I misunderstood. I think for MC events we need to use the obstime_mc in order to be able to create a posteriori a ra/dec for MC event in HA/dec as proposed @moralejo . To perform that, t I understood we wiil need:
```scale_time = sky_coords.obstime.scale
longitude = sky_coords.location.longitude
latitude = sky_coords.location.latitude
time=Time(obstime, scale, location=(longitude, latitude)
HA = sky_coords.icrs.ra.to("hourangle") -time.sidereal_time("apparent")
dec = sky_coords.icrs.dec
```

So we need to have a false obstime to be able to then transform to HA.

Reply:
I don't think @moralejo was proposing to actually calculate HA, just mentioning the possibility.

At the moment, there is no usecase I know of for having simulations in another frame than horizon.

A fake time however would never be the solution!

Reply:
I think that the monte carlo processing should be fully agnostic of any obstime. Introducing a fake obstime is a hack (that should be unnecessary).

If it at some position is necessary to introduce a fake time, I would be glad to know, because that means we nead it fixed.